Gemfile

```diff
@@ -11,7 +11,7 @@ gem "decidim-templates", DECIDIM_VERSION
 #### Custom gems and modifciations block start ####
 gem "decidim-admin-extended", path: "decidim-admin-extended"
 gem "decidim-challenges", "0.0.9", git: "https://github.com/gencat/decidim-module-challenges.git"
-gem "decidim-department_admin", "~> 0.4.0", git: "https://github.com/gencat/decidim-module-department_admin.git"
+gem "decidim-department_admin", git: "https://github.com/gencat/decidim-module-department_admin.git", tag: "v0.4.1"
```

Comment:
please, do not remove the version constraint, it helps many times to resolve the correct revision